### ISSUE TITLE
Default keyword inside switch block is not highlighted

### ISSUE THREAD

**Issue by kborowinski** [PowerShell/EditorSyntax#88]
### Environment
```powershell
> code-insiders.cmd -v
1.24.0-insider
d29620f2ba89327f51bb6fb2f05dfdd65e2fcafe

> $psEditor.EditorServicesVersion
Major  Minor  Build  Revision
-----  -----  -----  --------
1      7      0      0

> code-insiders.cmd --list-extensions --show-versions
...
ms-vscode.PowerShell@1.7.0
...

> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      5.1.17134.1
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17134.1
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

*Default* keyword inside *switch* block is not highlighted:

![image](https://user-images.githubusercontent.com/16168755/39743170-607d83a8-52a0-11e8-89df-bac65e706d28.png)

### Expected Behavior

*Default* keyword should be highlighted

**Comment by omniomi** [comment]
I took a quick peek and the switch block isn't defined so this isn't a quick fix. I've added it to my list and should be able to get it done tonight.

**Comment by omniomi** [comment]
Fixed by #94
